deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/config.json

```diff
@@ -42,7 +42,104 @@
       "Allow": true,
       "Except": null,
       "Publish": true,
```

Comment:
Note: ` "Publish": true,` was not changed so that new providers with different publishers would be indexed by default. We can change this to ` "Publish": false,` to block any publishers that are not explicitly allowed.